platform/core/src/services/ToolBarService/ToolbarService.ts

```diff
@@ -172,7 +172,13 @@ export default class ToolbarService extends PubSubService {
   }
 
   getActiveTools() {
-    return [this.state.primaryToolId, ...Object.keys(this.state.toggles)];
+    const activeTools = [this.state.primaryToolId];
```

Comment:
Somebody else broke this, and the generalized re-initialize of the toolbar needs this to work so fixing it here.